src/main/java/org/embulk/filter/column/ColumnFilterPlugin.java

```diff
@@ -117,7 +120,7 @@ static Schema buildOutputSchema(PluginTask task, Schema inputSchema)
                 boolean matched = false;
                 for (ColumnConfig dropColumn : dropColumns) {
                     // skip json path notation to build outputSchema
-                    if (dropColumn.getName().startsWith("$.")) {
+                    if (PathCompiler.isDocContext(dropColumn.getName())) {
```

Comment:
> the original implementation adds the dollar 

hmm. 

> In general, is prefix should be used for boolean variables and methods isn't it? 

It is correct, but I did not say anything about it.

How about having a method as below? It looks what we can tell is only "probably" json path.

```
    private static final String DOT_NOTATION_ROOT_PATH = new String(new char[]{DOC_CONTEXT, PERIOD});
    private static final String BRACKET_NOTATION_ROOT_PATH = new String(new char[]{DOC_CONTEXT, OPEN_SQUARE_BRACKET});

    public static boolean isProbablyJsonPath(String path)
    {
        return path.startsWith(DOT_NOTATION_ROOT_PATH) || path.startsWith(BRACKET_NOTATION_ROOT_PATH);
    }
```